config/routes.rb

```diff
@@ -17,6 +17,8 @@
   post 'questions/:id' => 'answers#create'
   post 'questions/:id/feedback' => 'questions#feedback'
 
+  get 'questions/:id/next' => 'questions#next'
```

Comment:
This bundle of work doesn't include the `next` and `previous` functionality, so can you remove this line before we commit it to the main project?